src/containers/App/reducer.js

```diff
@@ -29,16 +33,28 @@ export const initialState = fromJS({
   error: false,
   upload: {},
   userPermissions: [],
+  userName: undefined,
+  userScopes: undefined,
+  accessToken: undefined,
   categories: {
     main: [],
     sub: [],
     mainToSub: {},
-  }
+  },
 });
 
 function appReducer(state = initialState, action) {
   switch (action.type) {
+    case AUTHENTICATE_USER:
+      if (action.payload && action.payload.accessToken) {
+        global.sessionStorage.setItem(ACCESS_TOKEN, action.payload.accessToken);
```

Comment:
the global.sessionStorage is being set in the saga already: the login and logout functions in auth.js

Reply:
Good call. Removed it.